docs/source/configuring_zipkin.rst

```diff
@@ -84,6 +84,16 @@ zipkin.blacklisted_routes
         'zipkin.blacklisted_routes': ['some_internal_route',]
 
 
+zipkin.host
+~~~~~~~~~~~~~~~~~~
+    The host ip that the zipkin trace will include in its report.
```

Comment:
"report" is ambiguous. I'd use "span". Something like "the host ip that is used for spans" or something like that